modin/pandas/test/test_series.py

```diff
@@ -2183,7 +2201,14 @@ def test_mask():
 )
 def test_max(data, skipna):
     modin_series, pandas_series = create_test_series(data)
-    df_equals(modin_series.max(skipna=skipna), pandas_series.max(skipna=skipna))
+    try:
+        pandas_result = pandas_series.max(skipna=skipna)
+    except Exception as e:
+        with pytest.raises(type(e)):
+            modin_series.max(skipna=skipna)
+    else:
+        modin_result = modin_series.max(skipna=skipna)
+        df_equals(modin_result, pandas_result)
```

Comment:
```suggestion
    eval_general(*create_test_series(data), lambda df: df.max(skipna=skipna))
```

The same comments as for `test_all func`

Reply:
Fixed.